Data/Base/Shaders/Pipeline/SSAO.ezShader

```diff
@@ -38,16 +37,16 @@ Texture2D NoiseTexture;
 
 float SampleDepth(float2 texCoords)
 {
-  return DepthTexture.SampleLevel(DepthSampler, float3(texCoords, s_ActiveCameraEyeIndex), 0).r;
+  return DepthTexture.Load(int4(texCoords.x, texCoords.y, s_ActiveCameraEyeIndex, 0)).r;
 }
 
 float SampleLowResDepth(float2 texCoords, float mipLevel)
 {
 #if 0
-  return LinearizeZBufferDepth(DepthTexture.SampleLevel(DepthSampler, float3(texCoords, s_ActiveCameraEyeIndex), 0).r);
+  return LinearizeZBufferDepth(DepthTexture.Load(int4(texCoords.x, texCoords.y, s_ActiveCameraEyeIndex, 0)).r);
 #else
   texCoords = texCoords * TexCoordsScale;
-  return LowResDepthTexture.SampleLevel(DepthSampler, float3(texCoords, s_ActiveCameraEyeIndex), mipLevel).r;
+  return LowResDepthTexture.Load(int4(texCoords.x, texCoords.y, s_ActiveCameraEyeIndex, mipLevel)).r;
```

Comment:
Not sure what to do about this, WebGPU doesn't like sampling from depth textures (though may need to be investigated further).

Reply:
Again the coordinates are in the wrong space. Furthermore we use a border clamp sampler here so apart from the coordinate clamping we would also need to emulate the border behavior in shader which will be a waste of performance on platforms that support depth sampling. So I guess we would need a SAMPLE_DEPTH macro or function to abstract that.

Reply:
Removed from this PR.

Reply:
Replaced as well with `ezSampleLevel_PointClampBorder`.